### DISCUSSION TITLE
Deleted photos aren't removed from shared album

### DISCUSSION THREAD

**Discussion by Magneticdud** [ente-io/ente#3727] [General]
Disclaimer: I did a test on a self hosted instance so this might also be a problem with my S3 configuration (used garage, not minio)

I created three users, A, B and C

User A saves a picture on the server on an album.

User A shares that album with user B, user B sees that picture in their account.

Now, A deletes that picture, then empties the trash. That picture is no more shown in the album and no more shown in the trash.

User B can still see that picture in the shared gallery.

User A now shares the album with C

C can see the deleted picture even if wasn't present at the moment of sharing (so it's not a local cache issue)

B or C can't delete that picture as it belongs to A, but A doesn't have in the account.

Where is this picture now, did it go in a "shared" hidden trash can that will be emptied automatically in 30 days? Or it's like "orphaned" and stay shared with others forever?

**Comment by ua741** [comment]
Can you log out and log in again for User A and see if the picture is visible? Also, did you delete the picture using our mobile app?

**Reply by Magneticdud** [reply]
Yes, deleted from mobile app. Now that I logged with user A from web, I see the deleted picture.

I deleted the picture again, and now it appeared in user A trash can in mobile app without logging out from mobile app
